packages/app/components/Settings/CourseAdminPanel.tsx

```diff
@@ -159,38 +159,38 @@ export default function CourseAdminPanel({
           </Menu.Item>
         </Menu>
       </Col>
-      <VerticalDivider />
-      <Space direction="vertical" size={40} style={{ flexGrow: 1 }}>
-        <Col span={20}>
-          {currentSettings === CourseAdminOptions.EDIT_COURSE && (
-            <EditCourse courseId={courseId} />
-          )}
-          {currentSettings === CourseAdminOptions.CHECK_IN && (
-            <TACheckInCheckOutTimes courseId={courseId} />
-          )}
-          {currentSettings === CourseAdminOptions.ROSTER && (
-            <CourseRosterPage courseId={courseId} />
-          )}
-          {/* {currentSettings === CourseAdminOptions.ADD && (
+      <Col span={1}>
+        <VerticalDivider />
+      </Col>
```

Comment:
This looks ridiculous, I hope we can refactor with newer version of nextjs

Reply:
which? using Col with `span{1}`? That's a part of antd's Col component, it lets you specify the grid or whatever.